plugins/ingest-attachment/build.gradle

```diff
@@ -71,9 +71,9 @@ dependencies {
   api "org.apache.pdfbox:fontbox:${versions.pdfbox}"
   api "org.apache.pdfbox:jempbox:1.8.17"
   api "commons-logging:commons-logging:${versions.commonslogging}"
-  api "org.bouncycastle:bcmail-jdk15on:${versions.bouncycastle}"
```

Comment:
So pdfbox in `2.x` branch depends on `xxx-jdk15to18` artifacts, should we use those instead? That would be safer I think.